src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,172 @@
+
+### Engaging the Community of People With Disabilities
+
+- A co-op program within a design faculty has a specific stream established so that design students can be matched with community organizations focused on people with disabilities. Students work with the organizations and their clients to design and prototype new products targeted at the needs of people with disabilities.
+
+### Ed-Tech Hackathons and Code Sprints
+
+- An open-source OER platform holds a community hackathon involving accessiblity specialists, designers and developers to surface issues with the accessibility of the platform and prototype solutions to them. The results are used as part of the next year of project planning to prioritize addressing the platform's accessibility. -->
+
+## Tools
+
+The sections below include examples of tools (software, hardware and design-related) that can be useful in inclusive design hacking.
+
+### Tools From Prosperity 4All
+
+The <a href="http://www.prosperity4all.eu/" target="_blank" class="link-external">Prosperity 4All</a> project that Floe
+<a href="http://www.prosperity4all.eu/about-p4a/projectpartners/" target="_blank" class="link-external">participates in</a>has supported the development and cataloguing of many tools relevant to inclusive learning through hacking and making. These include:
+
+#### The Nexus
+
+The <a href="https://github.com/GPII/nexus" target="_blank" class="link-external">Nexus</a> is an integration technology based on the <a href="https://github.com/fluid-project/infusion" target="_blank" class="link-external">Infusion</a> Javascript framework that allows <a href="https://wiki.gpii.net/w/Nexus_API" target="_blank" class="link-external">distributed systems to be connected together through shared and transformed state over HTTP and WebSockets protocols, as described in detail here.</a>
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/7R_pz2Fz4qE" frameborder="0" allowfullscreen></iframe>
+
+#### AsTeRICS
+
+The <a href="https://github.com/asterics" target="_blank" class="link-external">Assistive Technology Rapid Integration &amp; Construction Set</a> is an open source construction set for assistive technologies.
+
+AsTeRICS can be used to rapidly prototype new forms of assistive technology using a graphical construction environment that runs on Windows or as a <a href="https://github.com/asterics/WebACS" target="_blank" class="link-external">web-based application.</a>
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/JCTK75cOqXk" frameborder="0" allowfullscreen></iframe>
+
+#### Inclusive Design Guide
+
+The <a href="https://guide.inclusivedesign.ca/" target="_blank" class="link-external">Inclusive Design Guide</a> is an evolving collection of insights, practices, tools and activities that can be applied a wide range of design challenges in the digital, physical and service realm.
```

Comment:
I think it might make sense to flesh this section out a bit, perhaps with the help of @dayotte. In particular, I'm thinking that we could describe how the Inclusive Design Guide could be used within the context of maker events and workshops to help catalyze the inclusive design process—as a tool for generating or evaluating ideas.